src/iuse_actor.cpp

```diff
@@ -2183,13 +2183,13 @@ void learn_spell_actor::info( const item &, std::vector<iteminfo> &dump ) const
                 const spell sp = pc.magic->get_spell( sp_id );
                 spell_text += ": " + string_format( _( "Level %u" ), sp.get_level() );
                 if( sp.is_max_level() ) {
-                    spell_text = "<color_light_green>" + spell_text + _( " (Max)" ) + "</color>";
+                    spell_text = string_format( "<color_light_green>%1$s %2$s</color>", spell_text, _( "(Max)" ) );
```

Comment:
```suggestion
                    spell_text = string_format( "<color_light_green>%1$s (Max)</color>", spell_text );
```

Reply:
I want translators to only translate “Max” word without the colour tags. If you merge them into one string then you need to translate it.

Reply:
That's a good point, but merging it into the string allows better flexibility as to how the translator translates the whole string. Maybe you can strip out the color tag and use `colorize_string` instead.